SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/DormancyTests/DormancyTest.cpp

```diff
@@ -6,7 +6,7 @@
 
 AActor* ADormancyTest::CreateDormancyTestActor()
 {
-	return GetWorld()->SpawnActor<ADormancyTestActor>({ 0.0f, 0.0f, 0.0f }, FRotator::ZeroRotator);
+	return SpawnActor<ADormancyTestActor>();
```

Comment:
Minor: would just do this inline at every call site, at this point this indirection is doing nothing more than making the code more opaque and indirect.

Reply:
added this